mothership-ec/cog-mothership-cms
# Page Tag Lazy Loading

#### What does this do?

Added lazy loading for page tags and switched content's lazy loading strategy to new one.
This also involved changing a few things in Union (https://github.com/messagedigital/union-music-store/pull/202).

I have not yet changed the `tags` array to be protected, because I was scared I might have missed a place somewhere. I have looked through all results in all directories (including vendor folders) for searching `page->tag` and couldn't find any other areas that use `tags` instead of `getTags`, but maybe someone else, who knows more about tags, could have a look at it too :)
#### How should this be manually tested?

Try to go through different type of pages, test on union, add new pages, add tags etc.
Have a look at the page objects and see whether they use the proxy etc. Use search to see if you can find any other areas in the system that have to be changed.
#### Related PRs / Issues / Resources?

Relies on https://github.com/messagedigital/cog/pull/386
Changes in Union: https://github.com/messagedigital/union-music-store/pull/202
#### Anything else to add? (Screenshots, background context, etc)
